packages/api/src/external/fhir/connector/connector.ts

```diff
@@ -9,3 +9,16 @@ export type FHIRServerRequest = {
 export interface FHIRServerConnector {
   upsertBatch(req: FHIRServerRequest): Promise<void>;
 }
+
+// TODO try to make `requestId` required
+export function makeJobId(requestId: string | undefined, documentId: string): string {
```

Comment:
nit: i thought we use `make` when creating objects for testing. Here, "createJobId" might be more suitable, similar to `createS3FileName`